src/datachain/cli/commands/ls.py

```diff
@@ -38,11 +38,12 @@ def ls_local(
 ):
     from datachain import DataChain
 
-    if catalog is None:
-        from datachain.catalog import get_catalog
-
-        catalog = get_catalog(client_config=client_config)
```

Comment:
Also `catalog` is only used if `sources` is set, so IMO no need to always init it.